pipeline-modules/app-service/aws/ecr-image-scan.yaml

```diff
@@ -0,0 +1,88 @@
+provisioner: aws
+name: ecr-image-scan
+version: 1
+revision: 1
+displayName: ECR Image Scan
+description: Run ECR Container Scan
+target: ""
+category: scan
+keywords:
+  - scan
+  - sast
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute the bash script from
+      type: string
+      default: repo
+    allow_critical:
+      title: Allow Critical Vulnerabilities
+      description: Allows builds with critical vulnerabilites
+      type: boolean
+      default: false
+    allow_high:
+      title: Allow High Vulnerabilities
+      description: Allows High vulnerabilites
+      type: boolean
+      default: false
+    allow_warning:
+      title: Allow Warning Vulnerabilities
+      description: Allows Warning vulnerabilites
+      type: boolean
+      default: false
+  internal:
+    - container_repository
+    - container_tag
+    - working_dir
+template: |
+  version: 0.2
+  env:
+    shell: bash
+  phases:
+    build:
+      commands:
+        - |
+          while:
+            do 
+              output=$(aws ecr describe-image-scan-findings --repository-name={{container_repository}}  --image-id imageTag={{container_tag}} --output json)
```

Comment:
Are we okh with using container_tag or sha_tag ?

Reply:
Are we okh with using container_tag or sha_tag ?

Reply:
container_tag is the generated one. It can be different then sha_tag if user did not select sha_tag.